frontstage/views/surveys/add_survey_confirm_organisation_survey.py

```diff
@@ -48,5 +48,6 @@ def survey_confirm_organisation(session):
     return render_template('surveys/surveys-confirm-organisation.html',
                            enrolment_code=enrolment_code,
                            encrypted_enrolment_code=encrypted_enrolment_code,
-                           organisation_name=organisation_name,
+                           organisation_name=business_party.get('name'),
+                           trading_as_name=business_party.get('trading_as'),
```

Comment:
So i think these should all go in to something like `busines_context` and then call on what you need in the template when you need it. Probably better standard that way, but I'll let you decide